test/inference/test_stablehlo.py

```diff
@@ -0,0 +1,49 @@
+import torch_xla
```

Comment:
you would need to add this test to `test/run_test.sh` otherwise it won't get run